avionics/common/include/scheduler_clock.hpp

```diff
@@ -0,0 +1,23 @@
+#ifndef SCHEDULER_CLOCK_HPP_7BA322CFFDDE4E3EB2CBA99E36951D6A
+#define SCHEDULER_CLOCK_HPP_7BA322CFFDDE4E3EB2CBA99E36951D6A
+
+#include "HAL/time.h"
+#include "RocketSchedule/schedule.hpp"
```

Comment:
Gives me a file not found error on compilation.

Reply:
Run `git submodule update` and try again

Reply:
@jesseli2002 Looks like you'll probably need to update the Tavis task in the GS to get the submodule https://stackoverflow.com/questions/3796927/how-to-git-clone-including-submodules

Or maybe better yet, you could add `git submodule update` as a build system step to cover the above case as well as ensure that people don't forget to update, build with the wrong version, and then experience a bug/inconsistency.

Reply:
https://stackoverflow.com/questions/4611512/is-there-a-way-to-make-git-pull-automatically-update-submodules/49427199

Seems like there's a way to get submodules to automatically update, with a git config thing. Travis commands on FLARE incorporate submodules - it's probably best to just copy what they use into the [.travis.yml in FLAREGUN](https://github.com/UBC-Rocket/UBCRocketGroundStation/pull/57)